### PR TITLE
Update documentation for new widgets/listing calls

### PR DESCRIPTION
**Pull request by OliverSkroblin**

### CODE REVIEW

File: source/developers-guide/attribute-system/index.md [hunk 1]
```diff
@@ -905,78 +905,56 @@ s_articles_attributes_my_column_helpText = "Deutscher help text"
 ```
 
 ## Product slider based on attributes
-__As of Shopware 5.3__ it is easily possible to create product slider with the help of attributes and the listing widgets:
-* `Shopware/Controllers/Widgets/Listing.php` - `streamSliderAction`
-* `Shopware/Controllers/Widgets/Listing.php` - `productSliderAction`
+__As of Shopware 5.3__ it is easily possible to display additional product sliders or box listing with the help of attributes and the listing widgets.
+The following functions allows to product templates based on a product stream or a selection of product numbers:
```
Comment (mitelg) [inline source/developers-guide/attribute-system/index.md]: allows to produce templates

don't know what you want to say here :blush:

File: source/developers-guide/attribute-system/index.md [hunk 2]
```diff
@@ -905,78 +905,56 @@ s_articles_attributes_my_column_helpText = "Deutscher help text"
...
+* `Shopware/Controllers/Widgets/Listing.php::streamAction`
+* `Shopware/Controllers/Widgets/Listing.php::productsAction`
 
-These can be used to create individual product slider. For example individual product slider for customers which will be shown in their account section. Here are two screenshots which illustrate that scenario:
+The following example displays additional products in a customer account. 
+First the s_user_attributes table get two new columns:
+- `recommendedVariants` - "Mehrfachauswahl - Varianten` - welche angezeigt werden sollen
```
Comment (mitelg) [inline source/developers-guide/attribute-system/index.md]: german?

File: source/developers-guide/attribute-system/index.md [hunk 3]
```diff
@@ -905,78 +905,56 @@ s_articles_attributes_my_column_helpText = "Deutscher help text"
...
+- `recommendedStream` - "Einfachauwahl - Product Stream" - welcher angezeigt werden soll
```
Comment (mitelg) [inline source/developers-guide/attribute-system/index.md]: german?

File: source/developers-guide/attribute-system/index.md [hunk 4]
```diff
@@ -905,78 +905,56 @@ s_articles_attributes_my_column_helpText = "Deutscher help text"
...
 
-<img src="img/customer.jpg" alt="customer details" class="image-border" width="800px" />
+<img src="img/customer.jpg" alt="customer details" class="image-border" />
 
-Choose a product stream and some variants for the customer with the help of attribute fields.
+In case of a plugin, use the follow code:
```
Comment (mitelg) [inline source/developers-guide/attribute-system/index.md]: use the following code

File: source/developers-guide/attribute-system/index.md (hunk 3 above)
Comment (OliverSkroblin) [inline source/developers-guide/attribute-system/index.md]: fixed

File: source/developers-guide/attribute-system/index.md (hunk 1 above)
Comment (OliverSkroblin) [inline source/developers-guide/attribute-system/index.md]: fixed

File: source/developers-guide/attribute-system/index.md (hunk 4 above)
Comment (OliverSkroblin) [inline source/developers-guide/attribute-system/index.md]: fixed

**Pull request merged**